integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSamples.java

```diff
@@ -122,86 +129,79 @@ public void testSampleDomainInPv(String model) {
-        + " -i " + Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString()
-        + " -o "
-        + Paths.get(sampleBase.toString()));
+    // run create-domain.sh to create domain.yaml file, run kubectl to create the domain and verify
+    createDomainAndVerify(domainName, sampleBase);
 
-    boolean result = Command.withParams(params).execute();
-    assertTrue(result, "Failed to create domain.yaml");
+    //delete the domain resource
+    deleteDomainResourceAndVerify(domainName, sampleBase);
+  }
 
-    // run kubectl to create the domain
-    params = new CommandParams().defaults();
-    params.command("kubectl apply -f "
-        + Paths.get(sampleBase.toString(), "weblogic-domains/" + domainName + "/domain.yaml").toString());
+  /**
+   * Test domain in image samples using domains created by wlst and wdt.
+   *
+   * @param model domain name and script type to create domain. Acceptable values of format String:wlst|wdt
+   */
+  @Order(2)
+  @ParameterizedTest
+  @MethodSource("paramProvider")
+  @DisplayName("Test samples using domain in image")
+  public void testSampleDomainInImage(String model) {
+    String domainName = model.split(":")[1];
+    String script = model.split(":")[0];
+    String imageName = (KIND_REPO != null
+        ? KIND_REPO + diiImageNameBase + "_" + script + ":" + diiImageTag
+        : diiImageNameBase + "_" + script + ":" + diiImageTag);
 
-    result = Command.withParams(params).execute();
-    assertTrue(result, "Failed to create domain custom resource");
+    //copy the samples directory to a temporary location
+    setupSample();
 
-    // wait for the domain to exist
-    logger.info("Checking for domain custom resource in namespace {0}", domainNamespace);
-    withStandardRetryPolicy
-        .conditionEvaluationListener(
-            condition -> logger.info("Waiting for domain {0} to be created in namespace {1} "
-                + "(elapsed time {2}ms, remaining time {3}ms)",
-                domainName,
-                domainNamespace,
-                condition.getElapsedTimeInMS(),
-                condition.getRemainingTimeInMS()))
-        .until(domainExists(domainName, DOMAIN_VERSION, domainNamespace));
+    Path sampleBase = Paths.get(tempSamplePath.toString(), "scripts/create-weblogic-domain/domain-home-in-image");
 
-    final String adminServerName = "admin-server";
-    final String adminServerPodName = domainName + "-" + adminServerName;
+    // update create-domain-inputs.yaml with the values from this test
+    updateDomainInputsFile(domainName, sampleBase);
 
-    final String managedServerNameBase = "managed-server";
-    String managedServerPodNamePrefix = domainName + "-" + managedServerNameBase;
-    int replicaCount = 2;
+    // update domainHomeImageBase with right values in create-domain-inputs.yaml
+    assertDoesNotThrow(() -> {
+      replaceStringInFile(Paths.get(sampleBase.toString(), "create-domain-inputs.yaml").toString(),
```

Comment:
I see these log messages in the test.out file which are confusing as its not actually copying the file, I think you can remove them or modify the log messages for string replacement ...https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2849/artifact/logdir/jenkins-weblogic-kubernetes-operator-kind-new-2849/wl_k8s_test_results/diagnosticlogs/ItSamples/ItSamples.out/*view*/
```
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <Copying /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <to /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <Copying /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <to /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <Copying /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <to /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <Copying /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <to /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <Copying /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <to /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <Copying /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
<10-30-2020 04:29:24> <INFO> <oracle.weblogic.kubernetes.utils.FileUtils replaceStringInFile> <to /tmp/it-results/sample-testing/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml>
```

Reply:
It's in fiuleUtils.java, do you want me change it there?

Reply:
yes please